stance/main.py

```
@@ -1,23 +1,25 @@
 import argparse
+import logging
+import pickle
 from etl import template_fit, etl
 from motion.squat import Squat
 from predict import score, overlay
-import pickle
 
-if __name__ == "__main__":
+
+def main():
```

If it's possible, can we throw an error if the user calls the main.py when they are not in stance/ since we have some files that require the main.py to be called when the user is in stance/? (Or we could change the filepath somehow but idk how to do that) ((I'm talking about squat_benchmark_zones.json in etl/))

---

(((Also the pickle files)))

---

Yeah so that's a fundamental problem with the way we do things. Rather than enforcing that the user has to be in a certain directory, our code should be independent of that.